apps/web/public/.well-known/apple-app-site-association

```diff
@@ -26,6 +26,7 @@
           "NOT /explore",
           "NOT /faq",
           "NOT /featured",
+          "NOT /features/*",
```

Comment:
couldn't test because localhost doesn't deep link, but this should work. can verify once we land